components/library/gd2/patches/000-cmake-soname.patch

```diff
@@ -0,0 +1,170 @@
+--- libgd-gd-2.2.4/CMakeLists.txt.orig	2017-06-10 01:26:44.907495342 +0000
```

Comment:
Basically I imported their patchset for soname from the 2.1 branch, I fixed the generation of pkg-config files with CMake and patched to keep the gdlib-config script (being deprecated) to keep proper detection by older components.